test/unit/annotation_test.rb

```diff
@@ -17,6 +17,16 @@ def setup
   end
 
   test 'geometry type' do
```

Comment:
Same as https://github.com/publiclab/mapknitter/pull/615/files#r289417232

Reply:
Should I wait for these last couple things before merging? Thanks!!!!